Makefile

```diff
@@ -5,11 +5,13 @@ all: vendor update test build
 include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	golang.mk \
 	targets/openshift/bindata.mk \
-	targets/openshift/crd-schema-gen.mk \
+	targets/openshift/controller-gen.mk \
 	targets/openshift/deps.mk \
 	targets/openshift/images.mk \
 )
 
+CONTROLLER_GEN_VERSION ?=v0.1.12
```

Comment:
I just noticed that this variable is not getting set here, since it was already set in targets/openshift/controller-gen.mk. So this is actually using controller-tools v0.2.1-37-ga3cca5d. The generated CRDs still look like they will be fine for OpenShift 3.11.